pywr/parameters/_parameters.pyx

```diff
@@ -905,6 +906,109 @@ cdef class UniformDrawdownProfileParameter(Parameter):
 UniformDrawdownProfileParameter.register()
 
 
+cdef class RbfProfileParameter(Parameter):
+    """Parameter which interpolates a daily profile using a radial basis function (RBF).
+
+    The daily profile is computed during model `reset` using a radial basis function with
+    day-of-year as the independent variables. The days of the year are defined by the user
+    alongside the values to use on each of those days for the interpolation. The first
+    day of the years should always be one, and its value is repeated as the 365th value.
+    In addition the second and penultimate values are mirrored to encourage a consistent
+    gradient to appear across the boundary. The RBF calculations are undertaken using
+    the `scipy.interpolate.Rbf` object, please refer to Scipy's documentation for more
+    information.
+
+    Parameters
+    ----------
+    days_of_year : iterable, integer
+        The days of the year at which the interpolation values are defined. The first
+        value should be one.
```

Comment:
Worth raising an exception if the first value is not 1?

Reply:
Good idea.

Reply:
Done.

Reply:
Actually, I guess this could be implemented without this being a requirement?